nomos-services/network/src/backends/libp2p/command.rs

```diff
@@ -21,6 +21,29 @@ pub enum Command {
         message: Box<[u8]>,
         retry_count: usize,
     },
+
+    // Kademlia commands
+    AddPeer {
+        peer_id: PeerId,
+        addr: Multiaddr,
+        reply: oneshot::Sender<Result<(), String>>,
+    },
+    FindPeer {
+        peer_id: PeerId,
+        reply: oneshot::Sender<Result<Vec<Multiaddr>, String>>,
+    },
+    PutValue {
+        key: Vec<u8>,
+        value: Vec<u8>,
+        reply: oneshot::Sender<Result<kad::RecordKey, String>>,
+    },
+    GetValue {
+        key: Vec<u8>,
+        reply: oneshot::Sender<Result<Vec<u8>, String>>,
+    },
```

Comment:
We have no use for the DHT other than peer discovery for now. @youngjoon-lee could we leverage this for the sync protocol?

Reply:
I have put the stubs just as ideas, so pls feel free to put your 2 c whoever thinks that some command could be useful. But also, new commands should be trivial to add, and I will reduce those to only what we use for now

Reply:
> We have no use for the DHT other than peer discovery for now. @youngjoon-lee could we leverage this for the sync protocol?

yeah, i guess that's a good approach to try (as we discussed today). I'll give a try and investigate some other approaches.